easybuild/easyconfigs/l/libsc/libsc-2.8-gmpolf-2017.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsc'
+version = '2.8'
+
+homepage = 'https://github.com/cburstedde/libsc'
+description = """ The SC Library provides support for parallel scientific
+applications."""
+
+toolchain = {'name': 'gmpolf', 'version': '2017.10'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/cburstedde/libsc/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['m4_files.patch']
+checksums = [
+    'fbfa61da6724c4606a8e93578404e7c935bc6371092e61dd369a81e207785dfa',  # v2.8.tar.gz
+    '604784abbb1d3b7ead4d8321048205f082d6a08002c4d3b7a98020cf38671d85',  # m4_files.patch
+]
+
+preconfigopts = 'echo "%(version)s" >> .tarball-version && ./bootstrap && '
+configopts = '--enable-mpi --enable-pthreads --enable-openmp'
+runtest = 'check'
+
+builddependencies = [
+    ('Autotools', '20170619')
+]
+
+sanity_check_paths = {
+    'files': ['bin/sc_b64dec', 'bin/sc_test_shmem',
+              'lib/libsc-%(version)s.so'],
```

Comment:
Please use `SHLIB_EXT` rather than hardcoding `.so`, as follows:

```python
sanity_check_paths = {
    'files': ['bin/sc_b64dec', 'bin/sc_test_shmem', 'lib/libsc-%%(version)s.%s' % SHLIB_EXT],